src/SourceCode.Clay.OpenApi/Serialization/OpenApiSerializer.cs

```diff
@@ -0,0 +1,1394 @@
+#region License
+
+// Copyright (c) K2 Workflow (SourceCode Technology Holdings Inc.). All rights reserved.
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+
+#endregion
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Json;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Net.Mail;
+using System.Net.Mime;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+
+namespace SourceCode.Clay.OpenApi.Serialization
+{
+    /// <summary>
+    /// Represents a serializer that can convert between OpenAPI objects and JSON.
+    /// </summary>
+    public class OpenApiSerializer : IOpenApiSerializer
```

Comment:
Break this up into partial classes (in a separate PR if you want). This is way to big to groc